docs/UserGuide.md

```diff
@@ -39,96 +39,166 @@
 Format: `help`
 
 > Help is also shown if you enter an incorrect command e.g. `abcd`
+
+Example:
+* `help` 
  
-#### Adding a person: `add`
-Adds a person to the address book<br>
-Format: `add NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...` 
+#### Adding a floating task: `add`
```

Comment:
Is the `floating task` the best name to use here? It's not a commonly used term.